### ISSUE TITLE
Layers might be slightly broken

### ISSUE THREAD

**Issue by acbart** [platipy/spyral#62]
A student reported that they had to define the .layers property before they could set .layer property. Need to repro and confirm, but I wouldn't be surprised considering how under-tested layers are.

**Comment by icarito** [comment]
makes sense as .layers belongs to Scene and .layer belongs to Sprite. It _should_ error out if you are defining a layer that doesn't exist, or at the very least if your scene has no layers defined at all. I'd close this one.

**Comment by acbart** [comment]
That's probably a reasonable restriction - I thought it would be convenient to let them write them in any order, but the layering system is a little delicate and probably shouldn't be too trifled with.
